_includes/payment-order-other-features.md

```diff
@@ -1554,19 +1565,6 @@ and Jekyll upgrades to it, we should also upgrade and replace all `include` with
 
 {% if include.documentation_section == "payment-menu" %}
```

Comment:
Should this `if` be deleted alongside its corresponding `endif`? The build is currently failing because they are not paired properly.

Reply:
Ah, I was looking for that one. Will remove it.